### ISSUE TITLE
puppet-gitlab requires outdated module dependencies

### ISSUE THREAD

**Issue by llauren** [voxpupuli/puppet-gitlab#152]
## General apology

I'm no expert in puppet nor git issues. Please be kind with my issue and me.

## Affected versions

* vshn/puppet-gitlab 1.15.1
* puppetlabs/apt 4.1.0
* Puppet 4.10.5
* Ubuntu 16.04 LTS

## How to reproduce

`puppet module install vshn-gitlab` when current version of `puppetlabs-apt` is installed, or

`puppet module list --verbose` when modules are installed by `--force`.

## What am i seeing?

```
$ puppet module list --verbose
Warning: Module 'puppetlabs-apt' (v4.1.0) fails to meet some dependencies:
  'vshn-gitlab' (v1.15.1) requires 'puppetlabs-apt' (>=2.1.0 <3.0.0)
...
```

## Expected behaviour

A clean install.

## Suggest solution

Bump required maximum version of `puppetlabs-apt` to `<5.0.0`. Unfortunately i don't have the competency to actually tell whether there is code that isn't 4.x-compliant.

**Comment by minorOffense** [comment]
According to the changelog in the apt module it should be fully backwards compatible through the 4.x release. 

https://github.com/puppetlabs/puppetlabs-apt/blob/master/CHANGELOG.md

Might just be worth changed the max version to less than 5 and see if any warnings appear in tests.

**Comment by minorOffense** [comment]
See https://github.com/vshn/puppet-gitlab/pull/169

**Comment by juniorsysadmin** [comment]
Closed by #169
